org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/window/WRecordIDDialog.java

```diff
@@ -219,14 +231,19 @@ else if(event.getTarget().equals(cancelBtn)) {
 	@Override
 	public void valueChange(ValueChangeEvent evt) {
 		if (evt.getSource().equals(tableIDEditor)) {
+			int tableID = Integer.parseInt(Objects.toString(evt.getNewValue(), "-1"));
+			MTable table = MTable.get(tableID);
```

Comment:
I think this is not safe, table can be null here.

Reply:
done